lib/engines/content_block_manager/app/models/content_block_manager/content_block/schema/embedded_schema.rb

```diff
@@ -40,6 +40,20 @@ def permitted_params
           end
         end
 
+        def govspeak_enabled?(field:, nested_object_key: nil)
```

Comment:
I wonder if this makes more sense to sit in the `Schema::Field` class?

Reply:
Actually, scratch that, as we don't have access to the `Field` object here. With that in mind, it might be better to call the `field` argument `field_name` to make it clearer it's a string we're passing in. Oh! My kingdom for a type checker!

Reply:
👍